GLMakie/src/drawing_primitives.jl

```diff
@@ -196,12 +202,13 @@ function handle_view(array::Observable{T}, attributes) where T <: SubArray
     return A
 end
 
-function lift_convert(key, value, plot)
-    return lift_convert_inner(value, Key{key}(), Key{Makie.plotkey(plot)}(), plot)
+function lift_convert(key, value, plot, screen)
+    return lift_convert_inner(value, Key{key}(), Key{Makie.plotkey(plot)}(), plot, screen)
 end
 
-function lift_convert_inner(value, key, plot_key, plot)
+function lift_convert_inner(value, key, plot_key, plot, screen)
     return lift(plot, value) do value
+        screen.requires_update = true
         return convert_attribute(value, key, plot_key)
     end
 end
```

Comment:
As far as I can tell all plot update tracking comes from this now? In that case I think it misses updates to camera matrices and lighting, because those get added to a render object from scene data. Though those should probably also be handled in screen.jl?

Reply:
Good point, didn't notice that, since enough is usually going on that it seems to be enough for most plots to rely on those updates...
Yes, I think we should add that here:
https://github.com/MakieOrg/Makie.jl/blob/master/GLMakie/src/screen.jl#L438